Source/Mosa.Platform.x64/CPURegister.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
+using Mosa.Compiler.Framework;
+using Mosa.Platform.Intel;
+
+namespace Mosa.Platform.x64
+{
+	public static class CPURegister
+	{
+		#region Physical Registers
+
+		/// <summary>
+		/// Represents the EAX register.
+		/// </summary>
+		public static readonly PhysicalRegister EAX = new PhysicalRegister(0, 0, "EAX", true, false);
```

Comment:
X64 registers use `R` instead of `E`, eg. `RAX`, `RCX`, `RDX`, etc. Best practice would be to follow that convention.

Reply:
Works for me!